openshift-install-powervs

```diff
@@ -1328,6 +1363,26 @@ function process_varfile {
   done
 }
 
+#-------------------------------------------------------------------------
+# Flavor
+#-------------------------------------------------------------------------
+function process_flavor {
+       flavor_file="$flavortype.tfvars"
+        [[ -z "$flavortype" ]] &&  error "The -flavor '$flavourtype' Please specify an option with flavor."
+
+       if [[ $flavortype == "small" || $flavortype == "medium" || $flavortype == "large" ]]; then
+          {
+              echo "IF_FLAVOR=2"
+              echo "FLAVOR_TYPE=$flavortype"
+          }  > logs/flavor.log
```

Comment:
No need to add it to logs.. logs are mostly TF console. You can use `log` function to display flavor on console.

Reply:
A file is used to retain the flavor specified. So that it can retrieve the flavor values in a different run. 
Example: 
$ ./openshift-install-powervs -flavor <option>
$ ./openshift-install-powervs variables